src/apps/btc/btc_sign.c

```diff
@@ -160,6 +164,52 @@ static void _maybe_pop_empty_screen(void)
     }
 }
 
+static void _maybe_pop_progress_screen(void)
+{
+    if (_progress_component != NULL) {
```

Comment:
>  It's always pushed at init 

Not if there is an `_error(...)` return before that in init(), on invalid input.

Also the client is can call any other function here instead of init, and run e.g. into this:

https://github.com/digitalbitbox/bitbox02-firmware/blob/74263e5b80c151e2a81040a174fb3411c4194cf3/src/apps/btc/btc_sign.c#L601

Reply:
> is no longer true

fixed!

Reply:
:+1: